x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -27,6 +27,9 @@ import { useApmParams } from '../../../../hooks/use_apm_params';
 import { isErrorMessage } from '../../correlations/utils/is_error_message';
 
 const DEFAULT_PERCENTILE_THRESHOLD = 95;
+// Enforce min height so it's consistent across all tabs on the same level
+// to prevent "flickering" behavior
+const MIN_TAB_TITLE_HEIGHT = 56;
```

Comment:
Why do the tabs change height in the first place?

Reply:
The small size `selection text` prompt is causing the width of that title box to be smaller than the other two (which have the standard size help button), so when you navigate between the tabs there's a bit of a "flickering" or shifting of the title/header spacer.